pymedphys/_dicom/delivery/core.py

```diff
@@ -66,7 +49,7 @@ def from_dicom(cls, dicom_dataset, fraction_number=None):
                     f"   Fraction numbers to choose from are: {fraction_numbers}"
                 )
 
-        beam_sequence, metersets = get_fraction_group_beam_sequence_and_meterset(
+        beam_sequence, metersets = rtplan.get_fraction_group_beam_sequence_and_meterset(
```

Comment:
I find it an order of magnitude easier to recognise an external function that has been defined in another file if it is instead imported and used this way. I am gradually converting pymedphys over to following this style.

It was recommended within the Google Style Guide.